src/config/manager.py

```
@@ -1,111 +1,156 @@
+        # Load all YAML files
+        for yaml_file in self.config_dir.glob("*.yaml"):
+            module = yaml_file.stem
+            if module not in self._cache:
+                self.load_config(module)
+                
+        return self._cache.copy()
 
 
-# Global config manager instance
-_config_manager = ConfigManager()
+import threading
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)